stac_model/model_metadata.py

```diff
@@ -0,0 +1,100 @@
+from pydantic import BaseModel, Field, FilePath, AnyUrl
+from typing import Optional, List, Tuple, Dict, Literal, Any
+from uuid import uuid4
+
+# Pydantic Models
+class TensorSignature(BaseModel):
```

Comment:
Needs to be adjusted for the new [Model Input Object](https://hackmd.io/DBRF1sQCS1WmSqygJNKQJQ?view#model-input-object) format.

Idem for output below.

Reply:
this is done in https://github.com/rbavery/dlm-extension/blob/fb1de05e8d0498c49b5e342583e7d21ca50e1981/stac_model/stac_model/__main__.py#L98-L99

I removed this old file.